pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -184,6 +184,10 @@ type PrometheusConfig struct {
 	// Enable Prometheus to be used as a receiver for the Prometheus remote write protocol. Defaults to the value of `false`.
 	// +optional
 	EnableRemoteWriteReceiver bool `json:"enableRemoteWriteReceiver,omitempty"`
+	// Enable Prometheus to accept OpenTelemetry Metrics via the otlp/http protocol.
+	// Defaults to the value of `false`.
+	// +optional
+	EnableOtlpHttpReceiver bool `json:"enableOtlpHttpReceiver,omitempty"`
```

Comment:
IIRC the prometheus OTLP endpoint is under some URL prefix. 

It might be good to document it here.

Reply:
I can do. Its `/api/v1/otlp`...`/v1/metrics`